### PR TITLE
Account theming

### PR DESCRIPTION
**Pull request by annachristaki**

### CODE REVIEW

File: keycloak/europeana-realm.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
   "loginWithEmailAllowed": true,
   "duplicateEmailsAllowed": false,
   "resetPasswordAllowed": true,
-  "editUsernameAllowed": false,
+  "editUsernameAllowed": true,
```
Comment (rwd) [inline keycloak/europeana-realm.json:32]: Let's make the europeana theme the default for this realm, with:
```diff
diff --git a/keycloak/europeana-realm.json b/keycloak/europeana-realm.json
index 6b1353c..221a221 100644
--- a/keycloak/europeana-realm.json
+++ b/keycloak/europeana-realm.json
@@ -600,6 +600,7 @@
     "from": "admin@keycloak"
   },
   "loginTheme": "europeana",
+  "accountTheme": "europeana",
   "eventsEnabled": false,
   "eventsListeners": [
     "jboss-logging"
```

**Review by rwd — CHANGES_REQUESTED** [review]

File: keycloak/europeana-realm.json (hunk 1 above)
Comment (annachristaki) [reply keycloak/europeana-realm.json:32]: You are right, forgot to include that:+1:

**Review by rwd — APPROVED** [review]

File: theme/account/password.ftl [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+            <div class="${properties.kcInputWrapperClass!}">
+                <input type="password" class="form-control" id="password-confirm" name="password-confirm" autocomplete="new-password">
+            </div>
+        </div>
+
+        <div class="form-group">
+            <div id="kc-form-buttons" class="${properties.kcInputWrapperClass!}">
+                <div class="form-buttons flex-row-reverse">
+                    <button type="submit" class="${properties.kcButtonClass!} ${properties.kcButtonPrimaryClass!} ${properties.kcButtonLargeClass!}" name="submitAction" value="Save">${msg("changePassword")}</button>
```
Comment (mirjamverloop) [inline theme/account/password.ftl:45]: Accoding to the design, the font-weight of the text should be 600. I also think that is consistent with buttons used elsewhere.

File: theme/account/template.ftl [hunk 3]
```diff
@@ -0,0 +1,85 @@
+<#macro mainLayout active bodyClass>
+<!doctype html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <meta name="robots" content="noindex, nofollow">
+
+    <title>${msg("accountManagementTitle")}</title>
+    <link rel="icon" href="${url.resourcesPath}/favicon.ico" />
```
Comment (mirjamverloop) [inline theme/account/template.ftl:10]: In the theme/login/template.ftl this path has been updated to `<link rel="icon" href="${url.resourcesPath}/img/favicon.ico" />`. Should it be updated here too?

File: theme/account/theme.properties [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+
+kcHtmlClass=
+kcLoginClass=
+
+
+kcLogoClass=login-pf-brand
+
+kcContainerClass=container-fluid
+kcContentClass=col-sm-8 col-sm-offset-2 col-md-6 col-md-offset-3 col-lg-6 col-lg-offset-3
```
Comment (mirjamverloop) [inline theme/account/theme.properties:21]: No need to add `col-lg-6 col-lg-offset-3`

File: theme/account/theme.properties [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+kcContentWrapperClass=row
+
+kcHeaderClass=login-pf-page-header
+kcFeedbackAreaClass=col-md-12
+kcLocaleClass=col-xs-12 col-sm-1
```
Comment (mirjamverloop) [inline theme/account/theme.properties:26]: `col-xs-12` doesn't exist, `col-12` does.

File: theme/account/theme.properties [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+kcAlertIconClasserror=pficon pficon-error-circle-o
+
+kcFormAreaClass=col-sm-10 col-sm-offset-1 col-md-8 col-md-offset-2 col-lg-8 col-lg-offset-2
```
Comment (mirjamverloop) [inline theme/account/theme.properties:29]: I don't think there is a need for adding `col-lg-8 col-lg-offset-2`.

File: theme/account/theme.properties [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+kcFormCardClass=card-pf
+#kcFormCardClass=card-ef
+
+kcFormCardAccountClass=login-pf-accounts
+kcFormSocialAccountClass=login-pf-social-section
+kcFormSocialAccountContentClass=col-xs-12 col-sm-6
```
Comment (mirjamverloop) [inline theme/account/theme.properties:35]: `col-xs-12 ` doesn't exist, `col-12` does.

**Review by mirjamverloop — CHANGES_REQUESTED** [review]

File: theme/account/theme.properties [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+kcResetFlowIcon=pficon pficon-arrow fa-2x
+kcWebAuthnKeyIcon=pficon pficon-key
+
+kcFormClass=form-horizontal
+kcFormGroupClass=form-group
+#kcFormGroupErrorClass=has-error
+kcFormGroupErrorClass=has-error
+
+kcLabelClass=control-label
+kcLabelWrapperClass=col-xs-12 col-sm-12 col-md-12 col-lg-12
```
[the remark this answers is not in the document]
Comment (annachristaki) [reply theme/account/theme.properties:58]: This file was copied over from the original Keycloak theme that served as the base for the custom europeana theme. The styles in theme.properties were predefined and not created during theme customisation (and also not checked for errors/redundancies). Very few of them are still used in the updated ftl files so maybe it's worth to do a cleanup.

File: theme/account/template.ftl (hunk 3 above)
Comment (annachristaki) [reply theme/account/template.ftl:10]: this is not used, I will remove it, favicon is set further down

**Review by mirjamverloop — APPROVED** [review]

**Pull request merged**